build.gradle

```diff
@@ -139,36 +151,62 @@ allprojects {
 	}
 
 	jar {
-		classifier = "dev"
+		archiveClassifier = "dev"
+	}
+
+	artifacts {
+		dev file: file("${project.buildDir}/libs/$archivesBaseName-${version}-dev.jar"), type: "jar", builtBy: jar
```

Comment:
Use
${project.version} instead of ${version}

Reply:
I don't see anything in the scope that provides a `version`, and `project` is a default scope offered for `build.gradle` script.

Reply:
Note that you can just have `$project.buildDir` and `$version` without the curly braces. (like `$archivesBaseName`)